### ISSUE TITLE
OpenCms 10: serialization policy files cannot be exported

### ISSUE THREAD

**Issue by frittinger** [alkacon/opencms-core#389]
Hi,

I have constantly errors about serialization policy files that cannot be exported and the default is used. So far I could not find problems other than the logged errors.

Apparently the files are part of the opencms-widget.jar but cannot be accessed via the JAR.

This error happens for a plain new OpenCms 10 demo setup.

```
25 Apr 2016 11:17:27,618 [http-apr-8080-exec-7] WARN  org.opencms.gwt.CmsGwtServiceContext  - ERROR: The serialization policy file '/cms/handleStatic/gwt/opencms/C2E676F52806BF5F39736A511DB5B2D9.gwt.rpc' was not found; did you forget to include it in this deployment?
25 Apr 2016 11:17:27,618 [http-apr-8080-exec-7] WARN  org.opencms.gwt.CmsGwtServiceContext  - Error reading resource from path "null".
org.opencms.file.CmsVfsException: Error reading resource from path "null".
	at org.opencms.db.CmsDbContext.throwException(CmsDbContext.java:339)
	at org.opencms.db.jpa.CmsDbContext.report(CmsDbContext.java:124)
	at org.opencms.db.CmsSecurityManager.readResource(CmsSecurityManager.java:4978)
	at org.opencms.file.CmsObject.readResource(CmsObject.java:3129)
	at org.opencms.file.CmsObject.readFile(CmsObject.java:2543)
	at org.opencms.file.CmsObject.readFile(CmsObject.java:2514)
	at org.opencms.gwt.CmsGwtServiceContext.getSerializationPolicy(CmsGwtServiceContext.java:249)
	at org.opencms.gwt.CmsGwtServiceContext.getSerializationPolicy(CmsGwtServiceContext.java:186)
	at org.opencms.gwt.CmsGwtService.doGetSerializationPolicy(CmsGwtService.java:276)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.getSerializationPolicy(RemoteServiceServlet.java:234)
	at com.google.gwt.user.server.rpc.impl.ServerSerializationStreamReader.prepareToRead(ServerSerializationStreamReader.java:505)
	at com.google.gwt.user.server.rpc.RPC.decodeRequest(RPC.java:240)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:296)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:373)
	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.opencms.gwt.CmsGwtService.service(CmsGwtService.java:194)
	at org.opencms.main.OpenCmsCore.invokeGwtService(OpenCmsCore.java:1701)
	at org.opencms.main.OpenCmsServlet.doGet(OpenCmsServlet.java:156)
	at org.opencms.main.OpenCmsServlet.doPost(OpenCmsServlet.java:175)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:217)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:616)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:518)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1091)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:673)
	at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.doRun(AprEndpoint.java:2503)
	at org.apache.tomcat.util.net.AprEndpoint$SocketProcessor.run(AprEndpoint.java:2492)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by haukeh** [comment]
Can confirm, lots of exceptions logged on WARN level....

**Comment by tHerrmann** [comment]
We fixed this issue with commit 716d083dc213d934263a96869d78635d25b76060. Will be included in the next release.

**Comment by tHerrmann** [comment]
This has been fixed in release 10.0.1.
